### PR TITLE
Update French local with actual words, domains, ans regions

### PR DESCRIPTION
**Pull request by Selenuix**
I commited somes changes for French local. I updated some files with acutal informations :)

### CODE REVIEW

File: src/Faker/Provider/fr_FR/Address.php [hunk 1]
```diff
@@ -110,7 +107,7 @@ public static function department()
     /**
      * Randomly returns a french department name.
      *
-     * @example 'Ardèche'
+     * @example 'Hauts-de-France'
```
Comment (fzaninotto) [inline src/Faker/Provider/fr_FR/Address.php]: Hauts de France is a region, not a department

File: src/Faker/Provider/fr_FR/Internet.php [hunk 2]
```diff
@@ -4,6 +4,10 @@
 
 class Internet extends \Faker\Provider\Internet
 {
-    protected static $freeEmailDomain = array('voila.fr', 'gmail.com', 'hotmail.fr', 'yahoo.fr', 'laposte.net', 'free.fr', 'sfr.fr', 'orange.fr', 'bouygtel.fr', 'club-internet.fr', 'dbmail.com', 'live.com', 'ifrance.com', 'noos.fr', 'tele2.fr', 'tiscali.fr', 'wanadoo.fr');
-    protected static $tld = array('com', 'com', 'com', 'net', 'org', 'fr', 'fr', 'fr');
+    protected static $freeEmailDomain = array(
```
Comment (fzaninotto) [inline src/Faker/Provider/fr_FR/Internet.php]: low value changes, please revert the changes in this file.

**Comment by pimjansen** [comment]
Bump! Could we please resolve?

**Comment by pimjansen** [comment]
I am going to close this PR as stale. If this still needs to be merged, please feel free to reopen it.
Thanks for contributing
